src/core/mount.c

```diff
@@ -1572,7 +1572,8 @@ static int mount_setup_new_unit(
          * loaded in now. */
         unit_add_to_load_queue(u);
 
-        *ret_flags = MOUNT_PROC_IS_MOUNTED | MOUNT_PROC_JUST_MOUNTED | MOUNT_PROC_JUST_CHANGED;
+        *ret_flags = MOUNT_PROC_DEVICE_CHANGED | \
```

Comment:
the "\" only makes sense in a macro, can be dropped here without ill effect

Reply:
Hehe, yes, I often get confused ...